source/includes/gcp/_routers.md

```diff
@@ -243,5 +243,23 @@ Add one or more BGP peers associated to existing router interface(s)
 | `bgpPeers[*].name`<br/>_string_          | The name of the BGP peer association.                                                        |
 | `bgpPeers[*].interfaceName`<br/>_string_ | The router interface through which this BGP peering will be realized.                        |
```

Comment:
Would you mind clarifying something for me, Shabir? Is "peering" the preferred industry word? 
Right now it doesn't quite sit well with me, but if it's the best way then it's the best way!

Reply:
@Gerudhoh That's right. To create a direct connect between an on=prem network and a cloud network you have to do a "Peering" between the edge devices. And that's a specific term.

Reply:
I now know my least favourite term 😛

Reply:
Hahahahah Lol!!!